### PR TITLE
Vectara Quickstart

### PR DESCRIPTION
**Pull request by sjyothi54**
**Ticket URL:** https://issues.newrelic.com/browse/NR-164852
**Description:** Created Vectara Quickstart

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Please use the SVG file format from the offical website - https://vectara.com/wp-content/uploads/2022/08/vectara-color-logo.svg

**Comment by sjyothi54** [comment]
> Please use the SVG file format from the offical website - https://vectara.com/wp-content/uploads/2022/08/vectara-color-logo.svg

@rahulbasu-nr Updated the logo, Kindly review this PR.

File: quickstarts/vectara/config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+slug: vectara
+description: |
+  ## Why should you monitor your usage of Vectara?
+  Monitor your vector searches on Vectara to get visibility on what you send to Vectara, responses retrieved from Vectara, latency, usage and errors. 
+  ### Track the query performance of your Vector DB
```
Comment (rahulbasu-nr) [inline quickstarts/vectara/config.yml:5]:
```suggestion
  ### Track the query performance of Vectara
```

File: quickstarts/vectara/config.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
+  ### Track your app:
```
Comment (rahulbasu-nr) [inline quickstarts/vectara/config.yml:7]:
```suggestion
  ### Track the health of Vectara
```

File: quickstarts/vectara/config.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+slug: vectara
+description: |
+  ## Why should you monitor your usage of Vectara?
+  Monitor your vector searches on Vectara to get visibility on what you send to Vectara, responses retrieved from Vectara, latency, usage and errors.
```
Comment (rahulbasu-nr) [inline quickstarts/vectara/config.yml:4]:
```suggestion
  Monitor your vector searches with Vectara to gain insights into the data you're sending to it, the responses you're receiving, the latency, usage, and any potential errors.
```

File: quickstarts/vectara/config.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+slug: vectara
+description: |
+  ## Why should you monitor your usage of Vectara?
+  Monitor your vector searches on Vectara to get visibility on what you send to Vectara, responses retrieved from Vectara, latency, usage and errors. 
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
+  ### Track your app:  
+  By tracking key metrics like latency, throughput, error rates, and input & output, you can gain insights into your app's performance and identify areas of improvement.
```
Comment (rahulbasu-nr) [inline quickstarts/vectara/config.yml:8]:
```suggestion
  By tracking key metrics like latency, throughput, error rates, and input & output, you can gain insights into Vectara's performance and identify areas of improvement.
```

File: quickstarts/vectara/config.yml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+  ### What’s included in this quickstart?
+  New Relic Vectara monitoring quickstart provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of your Vectara app. These reports include:
```
Comment (rahulbasu-nr) [inline quickstarts/vectara/config.yml:11]:
```suggestion
  New Relic Vectara monitoring quickstart provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of Vectara. These reports include:
```

File: quickstarts/vectara/config.yml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+  - Jyothi Surampudi
+title: Vectara
+documentation:
+  - name: Vectara integration documentation
+    description: |
+      Implement monitoring and instrumentation for your Vector store, and ensure that your observability data is integrated into New Relic for effective performance analysis and insights.
```
Comment (rahulbasu-nr) [inline quickstarts/vectara/config.yml:27]:
```suggestion
      Implement monitoring and instrumentation for your Vectara, and ensure that your observability data is integrated into New Relic for effective performance analysis and insights.
```

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Please make these changes.

**Comment by sjyothi54** [comment]
> Please make these changes.

Updated the changes

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Pull request merged**